ceno_zkvm/src/instructions/riscv/mulh.rs

```diff
@@ -307,8 +307,7 @@ impl<E: ExtensionField, I: RIVInstruction> Instruction<E> for MulhInstructionBas
                 let rs2_s = rs2_signed.assign_instance(instance, lk_multiplicity, &rs2_val)?;
                 rd_signed.assign_instance(instance, lk_multiplicity, &rd_val)?;
```

Comment:
I don't know why we are bothering with signs here: RiscV only has one MUL instruction for both signed and unsigned values, just like the only have one ADD and one SUB instruction, because the bit patterns are the same.